app/scripts/views/mixins/signin-mixin.js

```diff
@@ -98,6 +103,12 @@ define(function (require, exports, module) {
 
       return this.invokeBrokerMethod(brokerMethod, account)
         .then(this.navigate.bind(this, this.model.get('redirectTo') || 'settings', {}, navigateData));
+    },
+
+    engageForm() {
+      // user has engaged with the signin or sign up form
+      // log 'flow.engage' from both forms because both forms can login.
+      this.logEventOnce('flow.engage');
```

Comment:
Re: the name of this event, it is [currently specced](https://github.com/mozilla/fxa/tree/master/features/FxA-41-signin-funnel-metrics#detailed-design) differently to this. Now the spec can change, of course, but also I want to make sure that the event we emit is the one that is most useful for analysis purposes.

For flows that end at this point, where the user never submits the form, it might be useful to know which form the user was engaged with when they abandoned the flow. How tricky is it to change this event name so that it is `signup.engage` or `signin.engage`? What about `force_auth.engage`?